soroban-sdk/src/testutils.rs

```diff
@@ -416,3 +419,117 @@ pub trait Deployer {
     /// the provided address, or if the instance/code has expired.
     fn get_contract_code_ttl(&self, contract: &crate::Address) -> u32;
 }
+
+pub use xdr::AccountFlags as IssuerAccountFlags;
```

Comment:
Could probably shorten this without losing intent:
```suggestion
pub use xdr::AccountFlags as IssuerFlags;
```